<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>0</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>    VIDEO</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>0</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>      AUDIO</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="VigilButton" name="pushButton_2">
         <property name="text">
          <string>Qualité Image</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="VigilButton" name="pushButton_3">
         <property name="text">
          <string>Délai Voix</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="VigilButton" name="pushButton_1">
         <property name="text">
          <string>Coupures</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="VigilButton" name="pushButton_6">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Bruit ou Distorsion</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="VigilButton" name="pushButton_4">
         <property name="text">
          <string>Coupures</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="VigilButton" name="pushButton_5">
         <property name="text">
          <string>Echo</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VigilButton</class>
   <extends>QPushButton</extends>
   <header>vigilbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
